.github/workflows/lint.yml

```diff
@@ -23,6 +26,8 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: 12
+      - run: yarn
+      - run: yarn run run-linters
```

Comment:
We shouldn't use `yarn run run-linters` here, and `run-linters` isn't known to yarn. We are already running the linters with `lint-action` below.

Reply:
@git-bd1599 please fix

Reply:
fixed